### DISCUSSION TITLE
Use a mediawiki integrating code documentation, how-tos and other useful info

### DISCUSSION THREAD

**Discussion by cflaviu** [managarm/managarm#318] [Ideas and RFCs]
A mediawiki based site that would allow structured storage, collaboration and sharing of information about principles, architecture, code, protocols, how-tos and other useful info would be more efficient that editing special documentation files.

For example https://miraheze.org/ offers free mediawiki hosting without ads and even own domains could be used.

**Comment by Dennisbonke** [comment]
My preference would still go towards the current handbook.

**Reply by avdgrinten** [reply]
Agreed. The handbook is most likely easier to update since it is part of the repository. I would not be against a wiki for less formal information (such as how-tos) but I'd rather focus on the handbook for now as it is better to have one comprehensive source of information than multiple rather incomplete ones.

**Reply by cflaviu** [reply]
They could be used in parallel. Maybe the future will prove what is more preferable.

**Reply by Dennisbonke** [reply]
Imho we don’t have enough information to warrant a mediawiki. When we actually get a proper need for user guides and stuff we can reconsider. For now I agree with @avdgrinten that we should focus on improving existing documentation in the handbook

**Comment by ArsenArsen** [comment]
No, it's not worth the extra hosting effort and the fact it's a CMS is incredibly off-putting, especially when a system based on Git can be created more safely and effectively (consider SSGs like Hakyll, Jekyll, or indeed mdBook).

And, no, third party services are likely not a good solution either.
